From eb7f03c667261557d7f809f7851bad6b3eea4646 Mon Sep 17 00:00:00 2001
From: Mohammed Naser <mnaser@vexxhost.com>
Date: Mon, 8 Jan 2024 14:00:37 -0500
Subject: [PATCH] fix: Stop using batch_op for rename_table

For migrations that rename tables, batch_op is not needed, which
is also even causing issues with newer versions of Alembic.

Change-Id: Ib43f5a24c497f7d97cb2d852b99489b0c3bd75fb
---
 .../alembic/versions/5077ffcc5f1c_add_share_instances.py      | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/manila/db/migrations/alembic/versions/5077ffcc5f1c_add_share_instances.py b/manila/db/migrations/alembic/versions/5077ffcc5f1c_add_share_instances.py
index 42d26b75e..373e308d0 100644
--- a/manila/db/migrations/alembic/versions/5077ffcc5f1c_add_share_instances.py
+++ b/manila/db/migrations/alembic/versions/5077ffcc5f1c_add_share_instances.py
@@ -245,8 +245,8 @@ def upgrade_export_locations_table(connection):
     with op.batch_alter_table("share_export_locations") as batch_op:
         batch_op.drop_constraint('sel_id_fk', type_='foreignkey')
         batch_op.drop_column('share_id')
-        batch_op.rename_table('share_export_locations',
-                              'share_instance_export_locations')
+    op.rename_table('share_export_locations',
+                            'share_instance_export_locations')
 
 
 def downgrade_export_locations_table(connection):
-- 
2.34.1
